src/reporter.cc

```diff
@@ -71,6 +76,12 @@ void BenchmarkReporter::PrintBasicContext(std::ostream *out,
     Out << kv.first << ": " << kv.second << "\n";
   }
 
+  if (internal::global_context != nullptr) {
```

Comment:
I was actually expecting that `FLAGS_benchmark_context` would be parsed into `internal::global_context`.

Reply:
oh that would be smart.